crates/store/re_log_encoding/src/decoder/mod.rs

```diff
@@ -442,35 +520,41 @@ mod tests {
                 compression: Compression::LZ4,
                 serializer: Serializer::MsgPack,
             },
+            EncodingOptions {
+                compression: Compression::Off,
+                serializer: Serializer::Protobuf,
+            },
+            EncodingOptions {
+                compression: Compression::LZ4,
+                serializer: Serializer::Protobuf,
+            },
         ];
 
         for options in options {
+            println!("{options:?}");
```

Comment:
😬 

Tip: add a `// TODO` suffix on code you plan to remove (the linter will stop you from merging it).

Or use `dbg!(options);` (again, this will not pass CI)

You can also just use `re_log::trace` and run with `RUST_LOG=re_log_encoding=trace`

Reply:
I didn't plan to remove this, what's wrong with keeping a `println` in a test? It doesn't get printed unless the test fails, in which case it's useful information about which part of the test failed